extensions/arc/src/constants.ts

```diff
@@ -0,0 +1,98 @@
+	public static properties: IconPath;
+	public static networking: IconPath;
+
+	public static setExtensionContext(context: vscode.ExtensionContext) {
+		IconPathHelper.context = context;
+		IconPathHelper.add = {
+			light: IconPathHelper.context.asAbsolutePath('images/add.svg'),
+			dark: IconPathHelper.context.asAbsolutePath('images/add.svg')
+		};
+		IconPathHelper.edit = {
+			light: IconPathHelper.context.asAbsolutePath('images/edit.svg'),
+			dark: IconPathHelper.context.asAbsolutePath('images/edit.svg')
+		};
+		IconPathHelper.delete = {
+			light: IconPathHelper.context.asAbsolutePath('images/delete.svg'),
+			dark: IconPathHelper.context.asAbsolutePath('images/delete.svg')
+		};
+		IconPathHelper.openInTab = {
+			light: IconPathHelper.context.asAbsolutePath('images/open-in-tab.svg'),
+			dark: IconPathHelper.context.asAbsolutePath('images/open-in-tab.svg')
+		};
+		IconPathHelper.heart = {
+			light: IconPathHelper.context.asAbsolutePath('images/heart.svg'),
+			dark: IconPathHelper.context.asAbsolutePath('images/heart.svg')
+		};
+		IconPathHelper.copy = {
+			light: IconPathHelper.context.asAbsolutePath('images/copy.svg'),
+			dark: IconPathHelper.context.asAbsolutePath('images/copy.svg')
+		};
+		IconPathHelper.collapseUp = {
+			light: IconPathHelper.context.asAbsolutePath('images/collapse-up.svg'),
+			dark: IconPathHelper.context.asAbsolutePath('images/collapse-up-inverse.svg')
+		};
+		IconPathHelper.collapseDown = {
+			light: IconPathHelper.context.asAbsolutePath('images/collapse-down.svg'),
+			dark: IconPathHelper.context.asAbsolutePath('images/collapse-down-inverse.svg')
+		};
+		IconPathHelper.postgres = {
+			light: IconPathHelper.context.asAbsolutePath('images/postgres.svg'),
+			dark: IconPathHelper.context.asAbsolutePath('images/postgres.svg')
+		};
+		IconPathHelper.computeStorage = {
+			light: context.asAbsolutePath('images/billing.svg'),
+			dark: context.asAbsolutePath('images/billing.svg')
+		};
+		IconPathHelper.connection = {
+			light: context.asAbsolutePath('images/connections.svg'),
+			dark: context.asAbsolutePath('images/connections.svg')
+		};
+		IconPathHelper.backup = {
+			light: context.asAbsolutePath('images/migrate.svg'),
+			dark: context.asAbsolutePath('images/migrate.svg')
+		};
+		IconPathHelper.properties = {
+			light: context.asAbsolutePath('images/properties.svg'),
+			dark: context.asAbsolutePath('images/properties.svg')
+		};
+		IconPathHelper.networking = {
+			light: context.asAbsolutePath('images/security.svg'),
+			dark: context.asAbsolutePath('images/security.svg')
+		};
+	}
+}
+
+export namespace cssStyles {
+	export const text = { 'user-select': 'text', 'cursor': 'text' };
+	export const title = { ...text, 'font-weight': 'bold', 'font-size': '14px' };
+	export const tableHeader = { ...text, 'text-align': 'left', 'border': 'none' };
+	export const tableRow = { ...text, 'border-top': 'solid 1px #ccc', 'border-bottom': 'solid 1px #ccc', 'border-left': 'none', 'border-right': 'none' };
```

Comment:
is color #ccc good for all themes?: light, dark, high contrast

Reply:
Good question.  I'll have to test all the themes, but I copied this style from existing tables in ADS.

Reply:
It looks good to me in all the themes, though I don't know what the accessibility standard is.